engine-api/src/methods/get_payload.rs

```diff
@@ -159,7 +159,7 @@ mod tests {
                 "executionPayload": {
                     "parentHash": "0xe56ec7ba741931e8c55b7f654a6e56ed61cf8b8279bf5e3ef6ac86a11eb33a9d",
                     "feeRecipient": "0x4200000000000000000000000000000000000011",
-                    "stateRoot": "0xf3a022e9d83ae2c38348e3b726a1ec25a9d6a5c2a15913d187e8beaa1c3b5d7d",
+                    "stateRoot": "0x0e630edda6cec7ad91c29ba4573bca730cebcea96a7d100bc4d800f12f5a8140",
```

Comment:
Is this the right thing to do, to change the state root to what it returns when you run the test?

Reply:
Yes, the state root needs to be updated whenever we change the framework. This is the right thing to do.